gravitee-am-reporter/gravitee-am-reporter-kafka/src/main/java/io/gravitee/am/reporter/kafka/dto/AuditAccessPointDto.java

```diff
@@ -15,50 +15,27 @@
  */
 package io.gravitee.am.reporter.kafka.dto;
 
-public class AuditAccessPointDto {
-  private String id;
-  private String alternativeId;
-  private String displayName;
-  private String ipAddress;
-  private String userAgent;
-
-  public String getId() {
-    return this.id;
-  }
-
-  public void setId(String id) {
-    this.id = id;
-  }
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.Setter;
 
-  public String getAlternativeId() {
-    return this.alternativeId;
-  }
-
-  public void setAlternativeId(String alternativeId) {
-    this.alternativeId = alternativeId;
-  }
+@Setter
+@Getter
+public class AuditAccessPointDto {
 
-  public String getDisplayName() {
-    return this.displayName;
-  }
+    @JsonProperty
```

Comment:
I'm pretty sure that by default Jackson serializes everything it finds a public gettter/setter for - `@JsonProperty` isn't required unless you're customizing something